src/Microsoft.DotNet.Interactive.Http.Tests/HttpKernelTests.cs

```diff
@@ -2122,15 +2122,169 @@ public async Task when_response_is_slow_and_large_final_displayed_value_includes
             .FormattedValues.Single().Value.Should().ContainAll("Response", "Request", "Headers");
     }
 
-    [Fact(Skip = "Requires updates to HTTP parser")]
-    public void responses_to_named_requests_can_be_accessed_as_symbols_in_later_requests()
+    [Fact]
+    public async Task responses_to_named_requests_can_be_accessed_as_symbols_in_later_requests()
     {
         // Request Variables
         // Request variables are similar to file variables in some aspects like scope and definition location.However, they have some obvious differences.The definition syntax of request variables is just like a single-line comment, and follows // @name requestName or # @name requestName just before the desired request url. 
 
+        var client = new HttpClient();
+        using var kernel = new HttpKernel(client: client);
 
-        // TODO (responses_to_named_requests_can_be_accessed_as_symbols_in_later_requests) write test
-        throw new NotImplementedException();
+        using var _ = new AssertionScope();
```

Comment:
In this case you probably don't need the `AssertionScope`. Your first assertion seems like a kind of sanity check and there's no information being lost if the test stops there. `AssertionScope` is most useful when you have more than one assertion that can fail independently of one another, e.g. property checks on a single object.